bank_accounts.py

```diff
@@ -64,23 +82,27 @@ def create_bank_account():
 
         # Required params
         account_id = str(uuid4())
-        account_number = str(uuid4().int)[:6]
+        account_number = str(int(uuid4()))[:6]
```

Comment:
The approach i chose first gave me bunch of errors when running the test_app.py file, saying there's no "int" attribute to uuid4. I googled it, found that this might be the solution for it and eventually it gave me the desired result so i assumed this is the way to go.

Reply:
Look at the Faker library that can generate random data (e.g. name, email, etc) and you can use it to also generate a random account numbers:
```
from faker import Faker

fake = Faker()

# Generate a 10-digit account number
account_number = fake.numerify(text='##########')
print(account_number)
```